TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
+    user = 'X';
+    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+    messages = "Game - START!";
+    complete = false;
+
+    reset() {
+        this.moves = 0;
+        this.user = 'X';
+        this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+        this.messages = "Game - START!";
+        this.complete = false;
+    }
+
+    getMoves() {
+        return this.moves;
+    };
+
+    displayBoard() {
+        let text = '';
+        for (let i = 0; i < 3; i++) {
```

Comment:
`i` and `k` is a strange combination... I've seen more often `i` and `j`, `r` and `c`, `y` and `x`, or `m` and `n`...

I guess this is fine tho